envisage/tests/test_slice.py

```diff
@@ -89,7 +97,7 @@ def test_insert(self):
     def test_extend(self):
         """ extend """
 
-        self.f.l.append([99, 100])
+        self.f.l.extend([99, 100])
```

Comment:
Drive-by fix?

Reply:
Well, not *just* a drive-by fix: it's also necessary to make the tests pass. But it seems likely to me that a test called `test_extend` was actually intended to test `extend` rather than `append`.